doc/mgr/dashboard.rst

```diff
@@ -124,6 +124,67 @@ aspects of your Ceph cluster:
 * **Ceph Manager Modules**: Enable and disable all Ceph Manager modules, change
   the module-specific configuration settings.
 
+Dashboard Landing Page
+^^^^^^^^^^^^^^^^^^^^^^
+Displays overall cluster status, performance and capacity metrics. Gives instant
+feedback to changes in cluster and provides easy access to sub pages of dashboard.
+
+Status
+""""""
+
+* **Cluster Status**: Displays overall cluster health. 
+* **Hosts**: Display a list of all hosts associated to the cluster, which
```

Comment:
I'm sorry to intervene again, but I think these descriptions do not accurately describe what's visible on the landing page. Please limit these descriptions to explain the various widgets visible on the landing page only. It's fine to then add a comment that further information is available when clicking on the link that is active on several of the widgets.

Reply:
I tried following your comment. Please take a look and let me know what you think.

Reply:
Looking better, thanks! I left a number of textual improvement suggestions, please take them into consideration.